benchmarks/utilities/log_parser.hpp

```diff
@@ -164,7 +164,8 @@ inline std::vector<event> parse_csv(std::string const& filename)
 
   for (std::size_t i = 0; i < actions.size(); ++i) {
     auto const& action = actions[i];
-    RMM_EXPECTS((action == "allocate") or (action == "free"), "Invalid action string.");
+    RMM_EXPECTS((action == "allocate") or (action == "allocate failure") or (action == "free"),
```

Comment:
This will break the parsing as it assumes an event can only be an `action::ALLOCATE` or `action::FREE`. But adding an `allocate failure` effectively adds a third kind of event.

Reply:
Fixed.

Reply:
Adding a third event isn't quite what I meant. This parser is used for the replay benchmark as well and this will still break the replay benchmark.